ephios/templates/base.html

```diff
@@ -50,13 +50,12 @@
         {% block css %}{% endblock %}
     {% endcompress %}
 
-    {# as of 2021-06-14, the statici18n tag misbehaves with compression when 500 errors occur #}
```

Comment:
did you find out why it happened? did it stop happening now?

Reply:
so I ran a test with DEBUG=False and its all fine and generating a response with our 500.html template. As that template is not based on the `base.html` template anyway, there is no reason to fail now. (although it looks dull :sweat_smile: )